src/components/Pricing/experiment-control/Plans/index.tsx

```diff
@@ -109,17 +109,18 @@ const PricingTiers = ({ plans, unit, compact = false, type }) => {
                         title={
                             plans[0].free_allocation === up_to ? (
                                 'Free'
-                            ) : enterprise_flag_enabled && index === tiers.length - 1 ? (
-                                <s>
+                            ) : type === 'product_analytics' && !up_to ? (
+                                <>
                                     $
                                     {parseFloat(unit_amount_usd).toFixed(
                                         Math.max(
                                             ...plans[plans.length - 1].tiers.map(
                                                 (tier) => tier.unit_amount_usd.split('.')[1]?.length ?? 0
                                             )
                                         )
-                                    )}
-                                </s>
+                                    )}{' '}
+                                    - <Link to="/contact-sales">Contact us</Link> for custom pricing
```

Comment:
This just feels weird - like why would we show the price if custom pricing is available? And for whom is it available?

Eg. "Custom Enterprise pricing available for large volumes". Possibly just in an (i) tooltip after the final tier price with a Contact Us link (posthog.com tooltips can be interactive)

Reply:
<img width="1393" alt="Screenshot 2024-03-18 at 3 19 27 PM" src="https://github.com/PostHog/posthog.com/assets/21014901/6963ec6f-915e-48dd-b245-1847c8f84223">

Reply:
Yeah that's better I think but not a giant link, just link "get in touch" after 👍

Reply:
<img width="401" alt="Screenshot 2024-03-18 at 3 43 43 PM" src="https://github.com/PostHog/posthog.com/assets/21014901/e7c89ab1-9d85-4d01-9db7-a2ba18bfc95a">